resident/resident-service/src/main/java/io/mosip/resident/util/TemplateUtil.java

```diff
@@ -283,23 +301,19 @@ public  Tuple2<Map<String, String>, String> getAckTemplateVariablesForBookAnAppo
      }
 
     public Tuple2<Map<String, String>, String> getAckTemplateVariablesForUpdateMyUin(String eventId, String languageCode) {
-        Map<String, String> templateVariables = getCommonTemplateVariables(eventId);
-        ResidentTransactionEntity residentTransactionEntity = getEntityFromEventId(eventId);
-        templateVariables.put(TemplateVariablesConstants.PURPOSE, getPurposeFromResidentTransactionEntityLangCode(residentTransactionEntity,
-                languageCode));
-        templateVariables.put(TemplateVariablesConstants.AUTHENTICATION_MODE, residentTransactionEntity.getAuthTypeCode());
```

Comment:
Why the below two lines removed?

Reply:
These codes are already there in getCommonTemplateVariables so removed this.